--- libparserutils-0.2.1/Makefile.orig	2015-03-16 11:49:25.184170926 +0100
+++ libparserutils-0.2.1/Makefile	2015-03-16 11:49:37.534104020 +0100
@@ -15,10 +15,6 @@ TESTRUNNER := $(PERL) $(NSTESTTOOLS)/tes
 WARNFLAGS := -Wall -W -Wundef -Wpointer-arith -Wcast-align \
 	-Wwrite-strings -Wstrict-prototypes -Wmissing-prototypes \
 	-Wmissing-declarations -Wnested-externs -pedantic
-# BeOS/Haiku standard library headers create warnings.
-ifneq ($(BUILD),i586-pc-haiku)
-  WARNFLAGS := $(WARNFLAGS) -Werror
-endif
 
 CFLAGS := -D_BSD_SOURCE -I$(CURDIR)/include/ \
 	-I$(CURDIR)/src $(WARNFLAGS) $(CFLAGS)
